agent/config/beerocks_agent.conf.in

```diff
@@ -13,7 +13,7 @@ vendor=Intel
 model=prplMesh
 ucc_listener_port=8002 # 0 - disabled
 ucc_listener_slave_hostap_iface=wlan0
-enable_arp_monitor=1
+enable_arp_monitor=0
```

Comment:
That's not correct. The ARP monitor is used for discovering Ethernet clients (IP address is discovered by the DHCP monitor). AFAIK EasyMesh doesn't handle Ethernet clients, but we already have this built-in...

Reply:
In the commit message, I wrote that it observed to cause high CPU cycles and that this is a bandaid solution and better solution must be found. I also added a fixup that disable it only on RDKB.